### DISCUSSION TITLE
How to edit plate types in web app

### DISCUSSION THREAD

**Discussion by DanRothwell** [ukrobotics/DispenseLibDotNet#17] [Q&A] [answered]
(asking here because the web app doesn't have a feedback option)
If I have a custom plate type that isn't in the list, how do I add it? The options I get are only to select from the list or cancel.
![image](https://github.com/ukrobotics/DispenseLibDotNet/assets/111614151/7c711a2b-0228-4ed4-94ae-6a235b9c51e4)

**Comment by mcounsell** [comment, marked as answer]
We maintain the labware definitions on our public plate library here:

https://labware.ukrobotics.app/

Please contact me in person (or UK Robotics support ) with manf and part numbers of any plate types that you wish to add and we will update https://labware.ukrobotics.app/ within a few hours.
